eppo_client/config.py

```diff
@@ -8,6 +8,7 @@ class Config(SdkBaseModel):
     api_key: str
     base_url: str = "https://fscdn.eppo.cloud/api"
     assignment_logger: AssignmentLogger
+    is_graceful_mode: bool = True
```

Comment:
Just wondering, is this intended for customers to set? I.e. will this need an update in the docs?

Reply:
Yes they can set this, for example in dev mode